ontology/tss.ttl

```diff
@@ -0,0 +1,116 @@
+@prefix tss: <http://w3id.org/rdftss/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix cdt: <http://w3id.org/awslabs/neptune/SPARQL-CDTs/> .
+@base <http://w3id.org/rdftss/> .                                                 
+                                                                  
+<http://creativecommons.org/ns#license> a owl:AnnotationProperty .
+
+dcterms:contributor a owl:AnnotationProperty .
+
+dcterms:creator a owl:AnnotationProperty .
+
+dcterms:description a owl:AnnotationProperty .
+
+dcterms:issued a owl:AnnotationProperty .
+
+dcterms:modified a owl:AnnotationProperty .
+
+dcterms:publisher a owl:AnnotationProperty .
+
+dcterms:title a owl:AnnotationProperty .
+
+vann:preferredNamespacePrefix a owl:AnnotationProperty .
+
+vann:preferredNamespaceUri a owl:AnnotationProperty .
+
+<http://w3id.org/rdftss/> rdf:type owl:Ontology ;
+                          owl:versionIRI <http://w3id.org/rdftss/0.1.0> ;
+                          owl:versionInfo "0.1.0" ;
+                          dcterms:modified "2025-02-25" ;
+                          dcterms:description "Ontology for defining Time Series Snippets in RDF."@en ;
+                          dcterms:title "RDF Time Series Snippets"@en ;
+                          dcterms:contributor <https://dylannvanassche.be/#me>, <https://pietercolpaert.be/#me> ;
+                          vann:preferredNamespacePrefix "tss" ;
+                          vann:preferredNamespaceUri "http://w3id.org/tss" ;
```

Comment:
Let’s do https!

Reply:
Need to update then the spec as well.

Reply:
I don’t think the base IRI is in the spec yet

Reply:
s/update/add :)